### PR TITLE
SFR-1844: Re-instrument New Relic browser monitoring

### PR DESCRIPTION
**Pull request by jackiequach**
[Jira Ticket](http://jira.nypl.org/browse/SFR-1844)

### This PR does the following:
- Re-instrument New Relic browser monitoring using the copy/paste method for injecting the JS script.

### Testing requirements & instructions: 
-

### CODE REVIEW

**Comment by samanthaandrews** [comment]
Hey @jackiequach can you confirm that these changes work with the Docker build? You'll need to pass in the New Relic env vars on the command line in order to make sure New Relic is actually starting up. Asking because we recently deployed OEW to prod and realized a few hours later that the updates we had made to NR were causing an `Internal server error` that made us need to trigger a rollback.

**Comment by jackiequach** [comment]
@samanthaandrews The docker build check on this PR is passing: https://github.com/NYPL/sfr-bookfinder-front-end/actions/runs/7105319857/job/19342267475?pr=467

**Comment by samanthaandrews** [comment]
Yes! It was passing on OEW too. But the problem seemed to surface when we actually ran the app using the Docker build setup. So I'm hoping we can check it to confirm.

**Comment by jackiequach** [comment]
I ran the docker build locally with the NR build args and it is working as expected.

File: src/lib/newrelic/NewRelic.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+const defaultInfo: NRInfo = {
+  beacon: "gov-bam.nr-data.net",
+  errorBeacon: "gov-bam.nr-data.net",
+  licenseKey: "NRBR-75b5fdeaf978a4a39e8",
+  applicationID: "1447020806",
+  sa: 1,
+};
+export const devConfig: NRConfig = {
+  agentID: "1588857514",
+  applicationID: "1443695682",
```
Comment (samanthaandrews) [inline src/lib/newrelic/NewRelic.tsx:37]: woah i didn't know DRB had a dev, qa, and prod NR setup!

**Review by samanthaandrews — APPROVED** [review]

**Pull request merged**